app/helpers/application_helper.rb

```diff
@@ -7,4 +7,6 @@ module ApplicationHelper
 
   # OVERRIDE
   include Hyrax::HyraxHelperBehaviorDecorator
+  # OVERRIDE
```

Comment:
I think we usually try to include the version number from where the Override is from

Reply:
Yes, we usually do but the previous override didn't have it so I didn't add it.